product_secondary_unit/migrations/11.0.1.0.2/post-migration.py

```diff
@@ -0,0 +1,13 @@
+#  Copyright 2019 Tecnativa - Sergio Teruel
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    group_secondary_unit = env.ref(
+        'product_secondary_unit.group_secondary_unit')
+    users = env['res.users'].search([])
```

Comment:
Add `with_context(active_test=False)`

Reply:
Why?

Reply:
For writing the group also on eventually deactivated users (that can be activated later, and being without the group).

Reply:
Ok. Changes done